COMPOUNDING_RATES.md

```diff
@@ -0,0 +1,149 @@
+differences away for users and provide a single entry point to fixed and amplified yield lending on either of these
+protocols.
+
+## Compounding Token
+
+When supplying underlying to a lending protocol, one typically receives a token in exchange which can be redeemed at a 
+later time for the amount of underlying lent plus any interest generated over the lending period.
+
+Different lending protocols issue different tokens, e.g.
+
+- Compound: cToken (https://compound.finance/docs/ctokens)
+- Rari: fToken (https://docs.rari.capital/fuse/#ftoken-s)
+- Aave: aToken (https://docs.aave.com/developers/tokens/atoken)
+- Euler: eToken (https://docs.euler.finance/developers/contract-reference#ieuleretoken)
+- Yearn: yVault Token (https://docs.yearn.finance/getting-started/products/yvaults/vault-tokens)
+- ERC-4626: https://eips.ethereum.org/EIPS/eip-4626
+
+Swivel coins the term "Compounding Token" to refer to these tokens in general, emphasizing the notion that these tokens
+accrue (*compound*) interest over time. The mechanics behind each of the different compounding tokens might be different,
+but a common denominator is the idea, that compounding tokens can be exchanged at a later time for the amount of
+underlying lent plus a corresponding amount of accrued interest.
+
+One can understand a compounding token as a **share** of the total amount of underlying in a lending protocol's 
+pool/market.
+
+## exchangeRate
+
+As time progresses, the amount of underlying in a lending pool/market generally increases (due to interest payments
+from borrowers). This increase represents the interest generated by a pool/market and lending protocols distribute this
+interest in one way or another to the lenders - typically by means of establishing a **ratio** between the supplied 
+underlying (by lenders) and the total (ever increasing) amount of underlying in the pool/market.
+
+This ratio is often referred to as **exchange rate**. Generally speaking, the exchange rate establishes a 
+**price per share**. If a compounding token represents a share of the total assets in a pool/market, the exchange rate
+represents how much underlying a compounding token can be exchanged for at a given time.
+
+It is important to note, that different lending protocols express this ratio in different ways (they might scale an 
+`int` value by the amount of underlying token decimals or use a completely different scale to achieve higher precision),
+and Swivel does **not** attempt to normalize these scales.
+
+## interestRate
+
+In most cases, users are less interested in the current exchange rate of a lending protocol, but more interested in 
+projected annual percentage yield (APY) of a lending protocol, as this let's us compare the projected yield of a 
+lending protocol using a normalized and more familiar metric.
+
+A protocol's interest rate can usually be calculated from its exchange rate, by compounding the generated interest over
```

Comment:
Un sure if this is accurate, APY is from that "supplyRate" while exchangeRate is its own thing really.